### ISSUE TITLE
VAMC System Banner Alert w/ Situation Updates: Launch plan

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#14117]
## Description

Draft PR: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/14242

### Timing
Sprint 88 suggested, pending VHA DM sign off. Want to get it out before hurricane season ramps up

## Acceptance criteria

**Prep work**
- [x] I expect that PRs are merged to a common integration branch and tested before merging to main
    - [x]  merge #14048  PR when VHADM approves
    - [x]  merge #11331 PR when VHADM approves 
    - [x] merge #14181 PR 
- [x] I expect that KB articles have draft changes saved in Drupal and sent to VHA DM & Helpdesk for review
- [x] VHA DM has signed off on the character limit / limiting banners to one and overall CM plan, including KB updates in #14009
- [x] Helpdesk has signed off on KB articles - #14317
- [x] VHA DM sends email to editors 1 week before change - sending Wed 7/19

**Day of Launch**
- [x] **Wed 7/26** I expect that the integration branch PR is merged to main https://github.com/department-of-veterans-affairs/va.gov-cms/pull/14242.  (Tues confirmed with Michelle so changes will be in place first thing Wed am.)
- [x] After merge, before unpublishing existing banners: Spot check https://prod.cms.va.gov/admin/content/vamc-alerts-and-statuses for any new banners that are critical / recently published. 
    - [x] If any: raise @ mention to Michelle in DSVA slack
    - [x] With unpublish: Unpublish all but recent critical 
- [x] I expect that changes to the KB articles from #14009 are published
    - [x] [About VAMC alerts and operating statuses](https://prod.cms.va.gov/node/6868/revisions/view/721350/792314/visual_inline)
    - [x] [How to add a system banner alert](https://prod.cms.va.gov/node/6871/revisions/view/721349/792330/visual_inline)
    - [x] [Best practices for VAMC system banner alerts](https://prod.cms.va.gov/node/43819/revisions/view/656780/792554/visual_inline)
    - [x] [How to create a situation update](https://prod.cms.va.gov/node/23520/revisions/view/527774/791621/visual_inline)
- [x] Communicate with VHA DM about launch -- @mmiddaugh to communicate, either via email or Teams

**Comment by kmariepat-cityfriends** [comment]
Approval is not in place, expecting on 6/26

**Comment by kmariepat-cityfriends** [comment]
We discussed potential merge on July 5 due to risk and holiday, still need confirmation from VHA DM

**Comment by swirtSJW** [comment]
This Issue seems like a duplicate of https://github.com/department-of-veterans-affairs/va.gov-cms/pull/14242

**Comment by jilladams** [comment]
#14242 is the PR, this ticket tracks the merging of that PR so it's reflectable in sprint.

**Comment by jilladams** [comment]
This ticket did not complete in Sprint 88, as VHA DM has not yet sent the email. Next steps blocked on VHA DM decision making around comms. cc @davidmpickett @mmiddaugh

**Comment by jilladams** [comment]
Noting: In sprint planning we said: let's merge Tuesday for going into effect first thing Wed. 

However: in VHA DM sync right after planning, VHA DM comms will instead say: unpublishing will begin midafternoon on Wed 7/26, and editors can begin publishing new alerts on Thurs 7/27. SO: we will not merge until Wed 7/26. Updated in ticket body.

**Comment by jilladams** [comment]
@swirtSJW @omahane reopen, moving back to prod verification, since we have more steps to run for launch post-merge. (Note: another reason not to use "Closes: #xxxx".)

**Comment by swirtSJW** [comment]
This is the only Banner we had to edit and re-publish due to the fact that it was created today.
https://prod.cms.va.gov/va-southern-arizona-health-care/vamc-banner-alert/2023-07-26/green-valley-cboc-phone-outage

**Comment by omahane** [comment]
@mmiddaugh @jilladams We've unpublished the banners and published the newest revisions of the KB articles.

We are ready to reach out to VHA DM.

**Comment by jilladams** [comment]
Post deploy thread for ongoing status: https://dsva.slack.com/archives/C0FQSS30V/p1690400229384719

**Comment by jilladams** [comment]
Verified with VHA DM that we can consider our work complete here, and view has been provided to Justin & Blake (via #14247 , shared via https://trello.com/c/Tua3OO24/16-vamc-system-banner-alert-with-situation-updates with them for access.
